app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/index.js

```diff
@@ -321,31 +322,56 @@ const CurrentTrafficTable = ({
   const { appList, appLogs } = useMemo(() => getAppLogs(), [getAppLogs]);
   const { domainList, domainLogs } = useMemo(() => getDomainLogs(), [getDomainLogs]);
 
-  const getGroupLogs = () => {
-    const [logType, filter] = filterType?.split(" ") ?? [];
-    const logs = (filterType ? (logType === "app" ? appLogs[filter] : domainLogs[filter]) : requestLogs) || [];
-    const searchedLogs = getSearchedLogs(logs, searchKeyword);
-
-    return (
-      <GroupByNone
-        requestsLog={searchedLogs}
-        handleRowClick={handleRowClick}
-        emptyCtaText={emptyCtaText}
-        emptyCtaAction={emptyCtaAction}
-        emptyDesc={emptyDesc}
-      />
-    );
-  };
+  const getGroupLogs = useMemo(
+    () => () => {
+      const logs = Object.values(filterTypes).reduce((logs, filterType) => {
+        const [logType, filter] = filterType?.split(" ") ?? [];
+        return [...logs, ...(filter ? (logType === "app" ? appLogs[filter] : domainLogs[filter]) : [])];
+      }, []);
```

Comment:
Instead of keeping filters like this, can we use this schema instead?

```
{
    "domain": [],
    "app": [],
    "search": {
        "modifiers": ["regex", "insensitive"],
        "value": "xyz" 
    }
}
```